donation-core/src/main/java/com/donation/VideoServiceImpl.java

```diff
@@ -0,0 +1,98 @@
+package com.donation;
+
+import com.donation.crud.VideoCrudService;
+import com.donation.donor.model.Video;
+import org.reactivestreams.Subscriber;
+import org.reactivestreams.Subscription;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.web.servlet.mvc.method.annotation.StreamingResponseBody;
+import reactor.core.publisher.Flux;
+
+import java.io.*;
+import java.util.Arrays;
+
+@Service
+public class VideoServiceImpl implements VideoService {
+
+    @Autowired
+    VideoCrudService videoCrudService;
+
+    @Override
+    public StreamingResponseBody stream(Long idProject) throws FileNotFoundException {
```

Comment:
StreamingResponseBody should not be core...

Reply:
Done